conf/portal.application.conf

```diff
@@ -147,9 +147,11 @@ reporting {
     WEB_PAGE {
       "text/html" {
         type = "com.arpnetworking.metrics.portal.reports.impl.chrome.HtmlScreenshotRenderer"
+        chromePath = "" # will need to be changed to the actual Chrome executable path in order to use this renderer
```

Comment:
will this respect `$PATH`?

Reply:
Nope! This option must be given and it must point to the Chrom[e/ium] executable, or else an exception will get thrown whenever a web-based report tries to render.